src/EditorFeatures/Test/Diagnostics/DiagnosticDataTests.cs

```diff
@@ -125,7 +125,7 @@ private static async Task VerifyTextSpanAsync(string code, int startLine, int st
                 projectId: document.Project.Id,
                 customTags: ImmutableArray<string>.Empty,
                 properties: ImmutableDictionary<string, string>.Empty,
-                location: new DiagnosticDataLocation(document.Id, null, "originalFile1", startLine, startColumn, endLine, endColumn),
+                location: new DiagnosticDataLocation("originalFile1", document.Id, null, startLine, startColumn, endLine, endColumn),
```

Comment:
the file path is now required, so it moved from teh optional parameter section to the required section.